### PR TITLE
IA-2600 handle zero in select dropdown

### PR DESCRIPTION
**Pull request by quang-le**
Add checks to handle the case where the value is `0`

https://github.com/user-attachments/assets/a17b8902-1408-4218-8538-ff4aaa3366d2

### CODE REVIEW

File: src/components/inputs/Select/multi.js [hunk 1]
```diff
@@ -45,11 +45,13 @@ const MultiSelect = ({
     const { formatMessage } = useSafeIntl();
     const classes = useStyles();
     const shiftKeyIsDown = useKeyPressListener('Shift');
+    //  Handle numeric 0 as value
+    const hasValue = Boolean(value)|| value === 0
```
Comment (beygorghor) [inline src/components/inputs/Select/multi.js:49]: there is a missing white space here no ?

File: src/components/inputs/Select/single.js [hunk 2]
```diff
@@ -39,21 +39,23 @@ const SingleSelect = ({
 }) => {
     const { formatMessage } = useSafeIntl();
     const classes = useStyles();
+    //  Handle numeric 0 as value
+    const hasValue = Boolean(value)|| value === 0
```
Comment (beygorghor) [inline src/components/inputs/Select/single.js:43]: Same here ?

**Review by beygorghor — APPROVED** [review]

File: src/components/inputs/Select/multi.js (hunk 1 above)
Comment (quang-le) [reply src/components/inputs/Select/multi.js:49]: yeah, my linter seems to be on strike for this repo

File: src/components/inputs/Select/single.js (hunk 2 above)
Comment (quang-le) [reply src/components/inputs/Select/single.js]: same

**Pull request merged**